setup.py

```diff
@@ -369,18 +340,10 @@ def build_extension_cmake(self, ext):
                     "-DCMAKE_OSX_ARCHITECTURES={}".format(";".join(archs)),
                 ]
 
-        try:
-            cmake_command = os.environ.get("CMAKE_COMMAND", "cmake")
-            subprocess.run([cmake_command, *cmake_args], cwd=cmake_build_dir, check=True)
-            subprocess.run([cmake_command, "--build", ".", *build_args], cwd=cmake_build_dir, check=True)
-            subprocess.run([cmake_command, "--install", ".", *install_args], cwd=cmake_build_dir, check=True)
-        except subprocess.CalledProcessError as e:
-            print("WARNING: Command '{}' returned non-zero exit status {}.".format(e.cmd, e.returncode))
-            if not ext.permissive_build:
-                raise
-        except Exception:
-            print("WARNING: An error occurred while building the CMake extension.")
-            raise
+        cmake_command = os.environ.get("CMAKE_COMMAND", "cmake")
+        subprocess.run([cmake_command, *cmake_args], cwd=cmake_build_dir, check=True)
```

Comment:
Wondering if we could benefit from a `cmake` utility function to DRY this up a bit?

Reply:
That's a really good suggestion, but it would take me a while to clean this up properly.  Any complaints about doing this in a separate PR?